<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddEventDialog</class>
 <widget class="QDialog" name="AddEventDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>265</width>
    <height>238</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Choose the Event to Add</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="2" column="1">
      <widget class="QPushButton" name="pushButtonDraw">
       <property name="text">
        <string>Dra&amp;w</string>
       </property>
       <property name="icon">
        <iconset resource="../resources_editor.qrc">
         <normaloff>:/qtgameengine/icons/event-draw.png</normaloff>:/qtgameengine/icons/event-draw.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButtonMouse">
       <property name="text">
        <string>&amp;Mouse</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="pushButtonKeyRelease">
       <property name="text">
        <string>Ke&amp;y Release</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="pushButtonAlarm">
       <property name="text">
        <string>&amp;Alarm</string>
       </property>
       <property name="icon">
        <iconset resource="../resources_editor.qrc">
         <normaloff>:/qtgameengine/icons/event-alarm.png</normaloff>:/qtgameengine/icons/event-alarm.png</iconset>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="pushButtonStep">
       <property name="text">
        <string>&amp;Step</string>
       </property>
       <property name="icon">
        <iconset resource="../resources_editor.qrc">
         <normaloff>:/qtgameengine/icons/event-step.png</normaloff>:/qtgameengine/icons/event-step.png</iconset>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButtonCreate">
       <property name="text">
        <string>C&amp;reate</string>
       </property>
       <property name="icon">
        <iconset resource="../resources_editor.qrc">
         <normaloff>:/qtgameengine/icons/event-create.png</normaloff>:/qtgameengine/icons/event-create.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="pushButtonOther">
       <property name="text">
        <string>O&amp;ther</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="pushButtonCollision">
       <property name="text">
        <string>C&amp;ollision</string>
       </property>
       <property name="icon">
        <iconset resource="../resources_editor.qrc">
         <normaloff>:/qtgameengine/icons/event-collision.png</normaloff>:/qtgameengine/icons/event-collision.png</iconset>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="pushButtonDestroy">
       <property name="text">
        <string>&amp;Destroy</string>
       </property>
       <property name="icon">
        <iconset resource="../resources_editor.qrc">
         <normaloff>:/qtgameengine/icons/event-destroy.png</normaloff>:/qtgameengine/icons/event-destroy.png</iconset>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QPushButton" name="pushButtonKeyPress">
       <property name="text">
        <string>Key &amp;Press</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="pushButtonKeyboard">
       <property name="text">
        <string>&amp;Keyboard</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="pushButtonTrigger">
       <property name="text">
        <string>&amp;Trigger</string>
       </property>
       <property name="icon">
        <iconset resource="../resources_editor.qrc">
         <normaloff>:/qtgameengine/icons/trigger.png</normaloff>:/qtgameengine/icons/trigger.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel</set>
     </property>
     <property name="centerButtons">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resources_editor.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>AddEventDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>132</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AddEventDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>132</x>
     <y>196</y>
    </hint>
    <hint type="destinationlabel">
     <x>132</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
